File: Source/OldCode/Archive/2005 TVA Code Library/Version/TVACodeLibrary/TVA.Communication/Internal Code/GoodbyeMessage.vb
' 07-06-06

<Serializable()> _
Friend Class GoodbyeMessage

    Private m_id As Guid

    Public Sub New(ByVal id As Guid)
        Me.ID = id
    End Sub

    ''' <summary>
    ''' Gets or sets the disconnecting client's ID.
    ''' </summary>
    Public Property ID() As Guid
        Get
            Return m_id
        End Get
        Set(ByVal value As Guid)
            m_id = value
        End Set
    End Property

End Class

File: Source/OldCode/Archive/2005 TVA Code Library/Version/CS/TVA.Communication_CS/Internal Code/GoodbyeMessage.cs
using System.Diagnostics;
using System.Linq;
using System.Collections;
using Microsoft.VisualBasic;
using System.Collections.Generic;
using System;

// 07-06-06

namespace TVA.Communication
{
	[Serializable()]internal class GoodbyeMessage
	{
		
		
		private Guid m_id;
		
		public GoodbyeMessage(Guid id)
		{
			this.ID = id;
		}
		
		/// <summary>
		/// Gets or sets the disconnecting client's ID.
		/// </summary>
		public Guid ID
		{
			get
			{
				return m_id;
			}
			set
			{
				m_id = value;
			}
		}
		
	}
	
}
